superset/db_engine_specs/presto.py

```diff
@@ -1133,54 +1133,41 @@ def get_function_names(cls, database: "Database") -> List[str]:
         return database.get_df("SHOW FUNCTIONS")["Function"].tolist()
 
     @classmethod
-    def extract_errors(cls, ex: Exception) -> List[Dict[str, Any]]:
+    def extract_errors(cls, ex: Exception) -> List[SupersetError]:
         raw_message = cls._extract_error_message(ex)
 
         column_match = re.search(COLUMN_NOT_RESOLVED_ERROR_REGEX, raw_message)
```

Comment:
We can implement this error and `TABLE_DOES_NOT_EXIST_ERROR_REGEX` in `custom_errors`, and get rid of the `extract_errors` method here, like I did for Postgres. I didn't want to change it in this PR to keep it smaller.